hibernate-cache/src/main/java/org/infinispan/hibernate/cache/access/PutFromLoadValidator.java

```diff
@@ -586,6 +586,9 @@ public boolean beginInvalidatingWithPFER(Object lockOwner, Object key, Object va
 						continue;
 					}
 					long now = regionFactory.nextTimestamp();
+               if (trace) {
```

Comment:
Hmm, why? I've not changed anything in the rest of the method.

If you look at the files changed adding `?ts=3` at the end (e.g. [this URL](https://github.com/infinispan/infinispan/pull/5350/files?ts=3)), you won't have issues seeing where the changes happen.